integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwMiiDomain.java

```diff
@@ -358,4 +398,37 @@ private boolean patchDomainWithWalletFileSecret(String opssWalletFileSecretName)
 
     return patchDomainCustomResource(domainUid, fmwDomainNamespace, patch, V1Patch.PATCH_FORMAT_JSON_PATCH);
   }
+
+  private void restartOperator(String opNamespace) {
```

Comment:
Check if we have a utility method for this, if not create new Utility Class CommonOperatorUtils.java.  This can complement the JIRA for modularization  the common utilities.